tensorflow/lite/CMakeLists.txt

```diff
@@ -488,3 +489,8 @@ target_compile_options(_pywrap_tensorflow_interpreter_wrapper
   PUBLIC ${TFLITE_TARGET_PUBLIC_OPTIONS}
   PRIVATE ${TFLITE_TARGET_PRIVATE_OPTIONS}
 )
+
+IF(TFLITE_ENABLE_METAL AND CMAKE_SYSTEM_NAME STREQUAL Darwin)
+    enable_language(OBJCXX)
+    add_subdirectory(delegates/gpu)
```

Comment:
Could you put logics under "if(TFLITE_ENABLE_GPU)" block instead of create additional CMakeLists.txt?
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/CMakeLists.txt#L217

You might want to update TFLITE_DELEGATES_GPU_SRCS.

Reply:
 @terryheo 

> You might want to update TFLITE_DELEGATES_GPU_SRCS.

so just to confirm that I understand you right:

Do you mean adding `${TFLITE_SOURCE_DIR}/delegates/gpu/metal_delegate.mm` to  TFLITE_DELEGATES_GPU_SRCS so it become part of `libtensorflow-lite.a` or it still better to create independent `libmetal_delegate.a` - how it produced by bazel build?